tests/utils/runner.ts

```diff
@@ -19,6 +19,20 @@ export const graphDataTestRunner = async (
 
   const log_path = path.resolve(__dirname) + "/../logs/" + path.basename(logFileName).replace(/\.(ya?ml)|.(ts)$/, ".log");
   const graph = new GraphAI(graph_data, callbackDictonary);
+
+  graph.onLogCallback = ({ nodeId, state, inputs, result, errorMessage }) => {
+    if (state === "executing") {
```

Comment:
It's better to use something like NodeState.Executing

Reply:
Fixed.